src/components/data-processing/MitochondrialContent/CalculationConfig.jsx

```diff
@@ -45,7 +45,7 @@ const MitochondrialConfig = (props) => {
           max={10}
           value={config.methodSettings[activeMethod].binStep}
           onUpdate={(newValue) => updateSettingsForActiveMethod({ binStep: newValue })}
-          disabled={disabled}
+          disabled={disabled || plotType === 'logHistogram'}
         />
```

Comment:
can you also rename that file to match the component name, as this is the standard?

Reply:
What do you mean? which file?